modules/prediction/common/message_process.cc

```diff
@@ -90,15 +90,40 @@ void MessageProcess::OnPerception(
           AdapterConfig::PERCEPTION_OBSTACLES);
   CHECK(ptr_obstacles_container != nullptr);
 
-  ptr_obstacles_container->Insert(perception_obstacles);
+  // Insert ADC into the obstacle_container.
+  auto ptr_ego_pose_container =
+      ContainerManager::Instance()->GetContainer<PoseContainer>(
+          AdapterConfig::LOCALIZATION);
+  auto ptr_ego_trajectory_container =
+      ContainerManager::Instance()->GetContainer<ADCTrajectoryContainer>(
+          AdapterConfig::PLANNING_TRAJECTORY);
+  CHECK(ptr_ego_pose_container != nullptr &&
+        ptr_ego_trajectory_container != nullptr);
```

Comment:
Check them separately so you know which one causes failure if it really happens.

Reply:
@HongyiSun Merge this PR for now. Please make the modification in next commit.